File: Spring/Chapter4/java_demo/demo/src/main/java/com/aspectj_xml/XMLTest.java
package com.aspectj_xml;

import org.springframework.context.ApplicationContext;
import org.springframework.context.support.ClassPathXmlApplicationContext;

public class XMLTest {
    public static void test()
    {
        ApplicationContext context = new ClassPathXmlApplicationContext("applicationContext.xml");
        XMLInterface xmlInterface = (XMLInterface)context.getBean("xmlImpl");
        xmlInterface.delete();
        xmlInterface.modify();
        xmlInterface.save();
    }
}

File: Spring/Chapter4/kotlin_demo/demo/src/main/kotlin/aspectj_xml/XMLTest.kt
package aspectj_xml

import org.springframework.context.ApplicationContext
import org.springframework.context.support.ClassPathXmlApplicationContext

fun test()
{
    val context:ApplicationContext = ClassPathXmlApplicationContext("applicationContext.xml")
    val xmlInterface = context.getBean("xmlImpl") as XMLInterface
    xmlInterface.delete()
    xmlInterface.modify()
    xmlInterface.save()
}